File: webapps/smsc_cc/java/ru/novosoft/smsc/admin/protocol/Status.java
package ru.novosoft.smsc.admin.protocol;


class Status {

  Status(String name, byte status) {
    this.status = 0;
    this.name = "Undefined";
    this.status = status;
    this.name = name;
  }

  byte status = 0;
  String name = null;
}

File: src/mcisme/admin/messages/Status.cpp
#include "Status.hpp"

namespace smsc{
namespace mcisme{
namespace admin{
namespace messages{

const int8_t Status::OK;
const int8_t Status::SystemError;
const int8_t Status::SubscriberNotFound;
std::map<Status::type,std::string> Status::nameByValue;
std::map<std::string,Status::type> Status::valueByName;
Status::StaticInitializer Status::staticInitializer;

}
}
}
}
